benchmarks/bench_algos.py

```diff
@@ -0,0 +1,207 @@
+import pytest
+
+import cugraph
+from cugraph.tests import utils
+
+import pytest_benchmark
+# FIXME: Remove this when rapids_pytest_benchmark.gpubenchmark is available
+# everywhere
+try:
+    from rapids_pytest_benchmark import setFixtureParamNames
+except ImportError:
+    print("\n\nWARNING: rapids_pytest_benchmark is not installed, "
+          "falling back to pytest_benchmark fixtures.\n")
+
+    # if rapids_pytest_benchmark is not available, just perfrom time-only
+    # benchmarking and replace the util functions with nops
+    gpubenchmark = pytest_benchmark.plugin.benchmark
+
+    def setFixtureParamNames(*args, **kwargs):
+        pass
+
+
+# FIXME: write and use mechanism described here for specifying datasets:
+#        https://docs.rapids.ai/maintainers/datasets
+# FIXME: rlr: soc-twitter-2010.csv crashes with OOM error on my HP-Z8!
+UNDIRECTED_DATASETS = [
```

Comment:
How the csv files were generated? The mtx2csv script creates directed edges in both directions for undirected graphs (which is really a DiGraph). We should make sure that undirected data is stored there (ie. if we have `u v` there's no `v u` explicitly stored there). 
This will matter for reading time and symmetrize time.

Reply:
I'm currently using datasets from the `/datasets/afender` mount on the lab machines, and using CSV files that were already in place. I can look at the undirected CSVs to make sure both directions are present.